### PR TITLE
Add camera support Sony ILCE 6300 and A380 

### PR DESCRIPTION
**Pull request by schenlap**
I am the author of patches in #11060 and #10985

### CODE REVIEW

**Comment by LebedevRI** [comment]
Please split wb presets into separate pr, i will merge it. (also, you may want to add comment about firmware version before each camera presets)

As for raw support, matrix matches the one in dcraw, so i guess it is ok.
But did you actually check <Crop> values in gimp?

**Comment by schenlap** [comment]
I'll create pr's for wb changes, removed it from this pr now.
Yes, crop values are checked and ok

File: src/external/rawspeed/data/cameras.xml [hunk 1]
```diff
@@ -6249,6 +6249,17 @@
 		<Crop x="0" y="0" width="4599" height="3064"/>
 		<Sensor black="0" white="4095"/>
 	</Camera>
+	<Camera make="SONY" model="DSLR-A380">
+		<ID make="Sony" model="DSLR-A380">Sony DSLR-A380</ID>
```
Comment (LebedevRI) [inline src/external/rawspeed/data/cameras.xml]: You did check that the color matrix for this camera in src/external/adobe_coeff.c is the right one?
It is the same for 2 more cameras.

Comment (schenlap) [inline src/external/rawspeed/data/cameras.xml]: Yes they are the same. Output of tools/dngmeta.sh
`
DNG created by : Adobe DNG Converter 9.6 (Windows)
DNG Illuminant : 21 (should be 21)


$ nano -w src/external/adobe_coeff.c

{ "Sony DSLR-A380", { 6038,-1484,-579,-9145,16746,2512,-875,746,7218 } },
`

Comment (LebedevRI) [inline src/external/rawspeed/data/cameras.xml]: Okay.
I guess either all those cameras have exactly the same sensor and part of the firmware, or adobe f-ed up

**Pull request merged**